Gemfile

```diff
@@ -8,6 +8,8 @@ end
 gem "rails", "~> 5.1.2"
 gem "sqlite3"
 gem "puma", "~> 3.7"
+gem "sprockets", "3.7.1"
```

Comment:
Gem này em thêm vào khi import bootstrap bị lỗi (undefined method `bubbles'? for font-path...), em tham khảo trên GG thì có cao nhân hướng dẫn là cần bổ sung sprockets ở vị trí trước bootstrap-sass mới được (Nhưng sau khi thêm vào thì vẫn chưa được). Sau đó thì em phải thêm require: false cho gem "scss_lint" thì mới import đựoc bootstrap.